package-lock.json

```diff
@@ -35853,7 +35817,7 @@
         },
         "vendor/spryker/spryker/Bundles/Chart/assets/Zed": {
             "name": "spryker-zed-chart-plotly",
-            "version": "1.0.0",
+            "version": "0.0.1",
```

Comment:
We might have to revert this
Something went wrong here, maybe on the merge?

Reply:
waiting for @FRITS-Kh to check it

Reply:
My guess, dependabot (and the security issue that was supposed to get resolved in assets) did sth wrong here.

Reply:
The version is correct according to the [package.json](https://github.com/spryker/chart/blob/master/assets/Zed/package.json)  in the module

Reply:
Please see https://github.com/spryker-shop/suite/pull/1193/commits/b560c0ab380f7d55f41ae933e14ca24089a59b8a and what happened there
Thats the part that might need reverting.

Reply:
I don't know how the bot did it. 
But we're using just `npm install` command and `npm` updates packages according to their `package.json` version automatically.
We don't change package-lock manually.